Common/InternalFilesAssemblyResolver.cs

```diff
@@ -32,8 +34,14 @@ public InternalFilesAssemblyResolver(IEnumerable<string> directories)
         /// Gets the installation path of the .NET Framework.
         /// </summary>
         /// <value>The installation path of the .NET Framework.</value>
-        public static string DotNetFrameworkAssemblyPath { get; } = Path.GetDirectoryName(typeof(object).Assembly.Location);
-
+        public static string DotNetFrameworkAssemblyPath
+        {
+            get
+            {
+                return dotnetFrameworkVersion?.AssembliesPath;
+            }
+        }
+    
```

Comment:
Field can be removed and this property can be changed to '...DotNetFramewokAssemblyPath => DotNetFrameworkVersion.LatestVersionFromRegistry?.AssembliesPath;'

Reply:
updated, but kept using backing field as it would otherwise not be able to tellthe difference between not yet retrieved and retrieved but not found (and therefore would execute each time the property is accessed)